hot.go

```diff
@@ -49,6 +49,13 @@ func New(cfg *Config) (*Template, error) {
 		Out: os.Stdout,
 	}
 	tpl.Init()
+
+	if cfg.Dir == ""{
+		cfg.Dir, err := os.Getwd()
```

Comment:
@anonrose Thanks, this looks nice but unfortunate it breaks the package.
The `cfg.Dir` is already defined and so is `err` so maybe you would have done like this `cfg.Dir,err=os.Getwd()`.

I suggest you run go test after making the changes just to be sure the tests pass before resubmitting.